core-tests/shared/src/test/scala/zio/ZIOSpec.scala

```diff
@@ -766,7 +766,7 @@ object ZIOSpec extends ZIOBaseSpec {
           assert(unexpected, isEmpty) &&
           assert(result, isNone) // timeout happens
         }
-      },
+      } @@ flaky,
```

Comment:
Does this need a ticket?

Reply:
Opened a ticket for it here: https://github.com/zio/zio/issues/2243.